src/Logic.hs

```diff
@@ -428,18 +445,25 @@ handleCommentAdded triggerConfig projectConfig mergeWindowExemption prId author
             let (Config.MergeWindowExemptionConfiguration users) = mergeWindowExemption
             in elem user users
       isApproved <-
-        if isJust approvalType
+        if isSuccess approvalType
           then isReviewer author
           else pure False
       -- For now Friday at UTC+0 is good enough.
       -- See https://github.com/channable/hoff/pull/95 for caveats and improvement ideas.
       day <- dayOfWeek . utctDay <$> getDateTime
-      if isApproved
-        then -- The PR has now been approved by the author of the comment.
-         case fromJust approvalType of
-           -- To guard against accidental merges we make use of a merge window.
-           -- Merging inside this window is discouraged but can be overruled with a special command or by adding the
-           -- user to the merge window exemption list.
+      case approvalType of
+        -- The bot was not mentioned in the comment, ignore
+        Ignored -> pure state
+        -- The bot was mentioned but encountered an invalid command, report error and
+        -- take not further action
+        Unknown command -> do
+          () <- leaveComment prId ("`" <> command <> "` was not recognized as a valid command.")
```

Comment:
Not sure, but could this trigger a comment loop? Say I comment `@bot megre`, the comment will be included in the response and the `@bot` would trigger Hoff again. Could that happen or are we protected against that? Otherwise we can just reply that the last comment was not recognized, people will see what they typed.

Reply:
I think it should be alright because it uses `Text.isPrefixOf`, but just to be safe, I'll strip the prefix from `command` before putting it in the comment.